nextjs-app/app/page.tsx

```diff
@@ -90,11 +90,11 @@ export default function Home() {
         <GridItem as={Card} area="calendar" w="full" overflowX="auto">
           <CardHeader><Heading as="h3" size="sm">カレンダー</Heading></CardHeader>
           <CardBody>
-            <Box w="full" overflowX="auto">
+            <ScrollArea w="full" borderWidth={1}>
               <HStack w="full" gap={0}>
                 {
                   calendarMockData.map((data, index) => (
-                    <VStack key={index} h="full" flex={1} borderWidth={1} borderLeftWidth={index === 0 ? undefined : 0} p="md" minW="2xs">
+                    <VStack key={index} h="full" flex={1} borderRightWidth={index >= 0 ? '1' : '0'} p="md" minW="2xs">
```

Comment:
`index >= 0`の条件だと全ての要素に`borderRightWidth`が効いてしまうので以下のようにするのはどうでしょう？

配列の要素番号が最後以外の場合は`borderRightWidth`を有効にするようにしています。

```suggestion
                    <VStack key={index} h="full" flex={1} borderRightWidth={index < calendarMockData.length - 1 ? 1 : 0} p="md" minW="2xs">
```